app/views/results/marker/_keybinding.js.erb

```diff
@@ -8,12 +8,26 @@ Mousetrap.bind('>', function() {
   $('#student_selector>.right>a')[0].click();
 });
 
+// Go to the previous criterion with shift + up
+Mousetrap.bindGlobal('shift+up', function(e) {
```

Comment:
I'm not sure `bindGlobal` is a good idea here. There are other text boxes where I could see this shortcut conflicting. I'm assuming you put this in for the flexible criteria inputs - apparently if you add `class="mousetrap"` to the input, all bindings will go through. I think this is okay for the inputs?